test/test_cuda.py

```diff
@@ -22,6 +22,9 @@
 import torch
 import torch.cuda
 import torch.cuda.comm as comm
+from torch.cuda._memory_viz import profile_plot
+from torch.cuda._memory_viz import trace_plot
```

Comment:
Could you please add some brief comments in the function stubs about what profile_plot and trace_plot are? Also, a brief description of what this PR is changing the PR body would be helpful too.

Reply:
yep, added docstrings and PR comment.